cmd/neofs-node/config/tree/config.go

```diff
@@ -0,0 +1,77 @@
+package treeconfig
+
+import "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+
+const (
+	subsection = "tree"
+
+	// DefaultCacheSize is a default cache size of the tree service.
+	DefaultCacheSize = 10
+
+	// DefaultReplicationChannelCapacity is a default tree service
+	// replication channel capacity.
+	DefaultReplicationChannelCapacity = 64
+
+	// DefaultReplicationWorkerCount is a default tree service
+	// replication worker count.
+	DefaultReplicationWorkerCount = 64
```

Comment:
Let's not introduce defaults also here? We already have some in the service itself.

Reply:
ok, made them return 0